File: GpoSion-SPA/src/app/_models/movimiento-molde.ts
export interface MovimientoMolde {
  movimientoMoldeId: number;
  moldeId: number;
  molde: string;
  fecha: Date;
  estatusMoldeId: number;
  estatusMolde: string;
  observaciones: string;
  fechaCreacion: Date;
  creadoPor: string;
}

File: GpoSion.API/Models/MovimientoMolde.cs
using System;
using System.ComponentModel.DataAnnotations.Schema;

namespace GpoSion.API.Models
{
    public class MovimientoMolde
    {
        public int MovimientoMoldeId { get; set; }

        public int MoldeId { get; set; }

        [ForeignKey("MoldeId")]
        public virtual Molde Molde { get; set; }
        public DateTime Fecha { get; set; }

        public int EstatusMoldeId { get; set; }

        [ForeignKey("EstatusMoldeId")]
        public virtual EstatusMolde Estatus { get; set; }

        public string Observaciones { get; set; }

        public DateTime? FechaCreacion { get; set; }

        public string CreadoPorId { get; set; }

        [ForeignKey("CreadoPorId")]
        public virtual User CreadoPor { get; set; }

    }
}